Godeps/_workspace/src/github.com/openshift/source-to-image/pkg/api/types.go

```diff
@@ -91,6 +96,11 @@ type Config struct {
 	// Specify a relative directory inside the application repository that should
 	// be used as a root directory for the application.
 	ContextDir string
+
+	// AllowedUIDs is a list of user ranges of users allowed to run the builder image.
+	// If a range is specified and the builder image uses a non-numeric user or a user
+	// that is outside the specified range, then the build fails.
+	AllowedUIDs user.RangeList
```

Comment:
@pweil- Until we can get the range of users from SCCs, is this an acceptable approach to at least preventing the STI builder from running images that could potentially run as root?